config/environments/sandbox.rb

```diff
@@ -5,6 +5,8 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
 
+  config.middleware.use TimeTraveler
+
```

Comment:
Let's add it to deployed development too. That's what is on the dev environment and in review apps.

Also, let's duplicate the comment from `development` config, it makes sense here too :)

Reply:
Done

Reply:
Added additional check for non-production environment https://github.com/DFE-Digital/early-careers-framework/blob/CPDTP-371-create-header-to-control-dates/app/middlewares/time_traveler.rb#L9